### PR TITLE
Bluetooth shell

### PR DESCRIPTION
**Pull request by coder137**
- Added Bluetooth to CC1352R1 board
- Solves #7 Minor fixes to tinyprintf_overrides

### CODE REVIEW

File: cli_ftd/CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,9 @@ cmake_minimum_required(VERSION 3.17)
 # set(TARGET_BOARD "CC1352P_2_LAUNCHXL")
 set(TARGET_BOARD "CC1352R1_LAUNCHXL")
 
+set(UART_BAUD_RATE 9600) # 115200 (console) or 9600 (bluetooth)
```
Comment (ganeshram93) [inline cli_ftd/CMakeLists.txt:9]: So is this the only thing that needs to be changed in order to toggle between Bluetooth and console?

**Review by ganeshram93 — COMMENTED** [review]
Added 1 comment.

File: cli_ftd/user/tinyprintf/tinyprintf_override.c [hunk 2]
```diff
@@ -76,20 +76,21 @@ static void *tinyprintf_uart_task(void *arg) {
...
 
     bool recv = line_buffer__remove_line(&line_buffer, data, sizeof(data));
     if (!recv) {
-      usleep(TASK_DELAY);
+      usleep(TASK_DELAY_MICROSECONDS);
       continue;
     }
 
     // ! FIXME, We arent able to get a newline through the
     // `line_buffer__remove_line` api above
```
Comment (ganeshram93) [inline cli_ftd/user/tinyprintf/tinyprintf_override.c:90]: Can we remove these comments?

File: cli_ftd/user/tinyprintf/tinyprintf_override.c [hunk 3]
```diff
@@ -20,7 +20,7 @@
 // Tasks
 // This 1ms task delay is kept to avoid hogging the CPU
 // This allows other tasks to run
```
Comment (ganeshram93) [inline cli_ftd/user/tinyprintf/tinyprintf_override.c:22]: Update notes please

**Review by ganeshram93 — COMMENTED** [review]
Added additional comments.

File: cli_ftd/CMakeLists.txt [hunk 4]
```diff
@@ -6,6 +6,9 @@ cmake_minimum_required(VERSION 3.17)
...
+set(BLUETOOTH 1) # 1 or 0
```
Comment (coder137) [inline cli_ftd/CMakeLists.txt:10]: This is the variable you check for BLUETOOTH configurations

Add it to your debugPrint

i.e if DEBUG_PRINT == 1 && BLUETOOTH == 0 (only then can you debug print messages)

Comment (ganeshram93) [reply cli_ftd/CMakeLists.txt:10]: Got it.

File: cli_ftd/user/tinyprintf/tinyprintf_override.c (hunk 2 above)
Comment (ganeshram93) [reply cli_ftd/user/tinyprintf/tinyprintf_override.c:88]: Ignored.

File: cli_ftd/CMakeLists.txt (hunk 1 above)
Comment (ganeshram93) [reply cli_ftd/CMakeLists.txt:9]: Clarified.

**Review by ganeshram93 — APPROVED** [review]
Approved. Merge to Master.

File: cli_ftd/user/tinyprintf/tinyprintf_override.c (hunk 3 above)
Comment (ganeshram93) [reply cli_ftd/user/tinyprintf/tinyprintf_override.c]: Ignored.

**Pull request merged**